config/environment.js

```diff
@@ -30,6 +39,14 @@ module.exports = function (environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
+    ENV.acmidm = {
```

Comment:
if the env is dev you have to hardcode the values because there will be no values in what you just wrote